docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Write a Runtime Module as a Rust crate"
+---
+
+In this tutorial, you'll write a Substrate runtime module that lives in its own crate, and include it in a node based on the node-template. If you prefer you can follow along with the [video walkthrough](todo record after initial feedback).
+
+## Setup a Development Environment
+If you haven't already, follow the guide to [install substrate](getting-started/installing-substrate.md). You'll need the substrate-up scripts, but not the standard substrate node, so the `--fast` option is what you want here.
+
+## Create your node
+We'll begin by creating a brand new node template.
+```bash
+substrate-node-new modular-chain <your name>
+```
+If you've followed other runtime tutorials, you've probably started this way.
+
+In this tutorial, we're not going to write our module directly as part of the node template, but rather as a separate Rust crate. This approach provides a few advantages:
+1. Our module can be easily imported into other nodes in the future.
+2. Our module can be published to places like github or crates.io without publishing our entire node in the same place.
+
+We do still need the node template, however, to actually run our module.
+
+## Clone the Module Template Repository
```

Comment:
I suggest that you be generous with subsections in general. Makes it easier to comb through the docs and link to relevant pieces of information.